processor/lsmintervalprocessor/config/config.go

```diff
@@ -27,6 +27,10 @@ import (
 
 var _ component.Config = (*Config)(nil)
 
+const (
+	defaultMaxExponentialHistogramBuckets = 160
```

Comment:
Was it just to match with the spec - https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#base2-exponential-bucket-histogram-aggregation ? 

Can we add a link to call that out explicitly?